CHANGELOG.md

```diff
@@ -10,11 +10,50 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.31.0
+
+### Important Changes
+
+- **feat(node): Add `dataloader` integration (#13664)**
+
+This release adds a new optional integration for the [`dataloader` package](https://www.npmjs.com/package/dataloader).
```

Comment:
It's not optional - it's enabled by default: https://github.com/getsentry/sentry-javascript/blob/dbfd8e2d235a11ee2b59eee13386a0fa85e16495/packages/node/src/integrations/tracing/index.ts#L45

Reply:
oh thanks